inspirehep/dojson/hep/fields/bd90x99x.py

```diff
@@ -54,6 +57,11 @@ def get_value(value):
                 year = int(value.get('y'))
             except:
                 pass
+        if 'i' in value:
+            try:
+                isbn = normalize_isbn(value.get('i'))
+            except:
```

Comment:
Agree that we should try to avoid `except..pass`, but rather understand what might go wrong in this case. Say we can check if `value.get('i')` returns something tangible. Like a boolean test `if value.get('i'):`.